src/compass.js

```diff
@@ -46,6 +47,9 @@ function clearCompass() {
 
 function draw(what) {
   console.log("Call: Compass.draw");
+  compass.backgroundColor(Themes.currentTheme().textColour);
```

Comment:
reinit doesn't really make much sense as you don't create/re-hook any elements, perhaps update?

Ideally you'd be able to call back into the watch from the settings page but I don't know anything about clay so I have no idea if it's possible. It looks like it might be (https://github.com/pebble/clay#custom-function) but ¯\_(ツ)_/¯